scripts/globals/weaponskills.lua

```diff
@@ -1052,7 +1052,11 @@ xi.weaponskills.takeWeaponskillDamage = function(defender, attacker, wsParams, p
 
     local enmityEntity = wsResults.taChar or attacker
 
-    if wsParams.overrideCE and wsParams.overrideVE then
+    if
+        wsParams.overrideCE and
+        wsParams.overrideVE and
+        wsResults.tpHitsLanded > 0
```

Comment:
I think this last line should be
`wsResults.tpHitsLanded + wsResults.extraHitsLanded > 0`